slackbridge/utils.py

```diff
@@ -182,7 +129,7 @@ def emoji_replace(match: Match[str]) -> str:
     text = re.sub(r'<\@(U\w+)\|?(\w+)?>', user_replace, text)
     text = re.sub(r'<!(\w+)\|?(\w+)?>', var_replace, text)
     text = re.sub(r'<(?!!)([^|]+?)>', lambda match: match.group(1), text)
-    text = re.sub(r':([\d\w+-]+):', emoji_replace, text)
+    text = emojize(re.sub(r':([\d\w+-]+):', lambda match: match.group(1), text))
```

Comment:
Use `use_aliases=True` to catch the aliases that Slack uses.

Reply:
I think that here it would make the most sense to ditch the regex and run emojize on the whole text, so just `text = emojize(text)`

Reply:
Er, yeah, it probably must - the examples look like the library trafficks in whole text, so it probably doesn't even work like this

Reply:
Er, yeah, it probably must - the library's examples look like the library trafficks in whole text, so it probably doesn't even work like this